docs/workflows/_bind-passkey-backend-email.mdx

```diff
@@ -0,0 +1,106 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Get Binding Link for Identity
+
+In order to get a binding link for a passkey, we need to create a binding job for an existing identity. The following code snippet uses the `EMAIL` delivery method. This method will send your user an email with a link that, when clicked, redirects the user to your application and provides your application with a binding link. If you wish to configure email branding, visit the Admin Console. This binding link is the link your application will use to complete the passkey binding process.
```

Comment:
When I test generating an EMAIL url from postman, it looks like the user is redirected to a url like the below- so individual query string parameters vs a complete url. Is it necessary to reconstitute a url from the params, or otherwise how should the app invoke the SDK? 
302 to
http://localhost:8083/bind?api_base_url=https%3A%2F%2Fauth-us.beyondidentity.com&identity_id=34c6a08b9b900389&job_id=caaf4c7229bd5830&realm_id=6aae0d38c246f6c4&tenant_id=00018c7edb8e1e3c&token=dpw6ly1LXtgt571JRhudWyZ7UGwXpVNY2YXnWjYd0dGDm3bKKncp5bHUWfihbteh

Reply:
Hm not sure I understand the question. The Invoke URL is set in the Authenticator Config.  The Invoke URL "points" to where your application is. In the case of a web application, this is just a URL to your web application or a specific page of your web application.

When binding with email, the Invoke URL is used to redirect from the email client into your application. So, the only thing you need to set as a developer is the Invoke URL.

Reply:
I think what i was missing was what we send to bindCredential as url in this case, and i think i'm hearing that we send literally this entire url, such as the example above, app hostname and all. Is that true?
Also i'm still wondering where this mystery '/bind' is coming from as it is absent from my app's config, but i'm going to consider that a personal problem for now...

Reply:
Yea, you literally feed the url you get into the bindPasskey function on redirection back into the app from the email client. You could be redirected into your app for many reasons though (like other apps might link to your app) so we have the isBindPasskeyURL to confirm the the url you received is a url you should send to the bindPasskey function. As far as the mystery /bind that you see added to the url, that might be what Beyond Identity adds to the bind url. But I’m not sure, if have to check. Do you see that added to the url on both “return” and “email” binds?

Reply:
Ok Just tested this on acme and it looks like after tapping the email, that url gets redirected to the InvokeURL with an appended /bind. That’s the bind url that you would feed into the bindPasskey function. So if you wanted you could add a /bind route to your application to grab that bind url specifically. Am I missing anything @ismailmustafa ?

Reply:
I just added more info to this document. Let me know if it's clearer.

Reply:
yes, it is clearer to me now. it looks good - thanks!